UM/Operations/MirrorOperation.py

```diff
@@ -36,6 +36,7 @@ def redo(self):
             center = self._node.getPosition()
             self._node.setPosition(-center)
         self._node.scale(self._mirror, SceneNode.TransformSpace.World) #Then mirror around the origin.
+        self._node.invertNormals()  # Because the mirror is done via a scale operation, the normals where inverted.
```

Comment:
@rburema I guess `invertNormals()` should also happen in `undo()`?

Reply:
Added in 98d2380b

Reply:
Yes, thanks.